scripts/setup.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs')
+const path = require('path')
+
+/* 
+****************************
+Update package.json
+****************************
+*/
+
+const packageJSON = require('../package.json')
+const { name: projectName } = packageJSON
+
+// ensure that yarn init has been run
+if (projectName === undefined) {
+  console.log(
+    'Warning: no name defined in package.json. Please run "yarn init" before "yarn setup".',
+  )
+  process.exit(1)
+}
+
+// 1. Change "name" in the "jbrowse-plugin" field to the name of your project (e.g. "MyProject")
+packageJSON['jbrowse-plugin'].name = projectName
+
+// 2. In the "scripts" field, replace the default name with the name of your project, prefixed with "JBrowsePlugin" in the "start" and "build" entries
+packageJSON.scripts.start = `tsdx watch --verbose --noClean --format umd --name JBrowsePlugin${projectName} --onFirstSuccess "yarn serve --cors --listen 9000 ."`
+
+packageJSON.scripts.build = `tsdx build --format cjs,esm,umd --name JBrowsePlugin${projectName}`
+
+// 3. In the "module" field, replace jbrowse-plugin-my-project with the name of your project (leave off the @myscope if using a scoped package name) (you can double-check that the filename is correct after running the build step below and comparing the filename to the file in the dist/ folder)
+packageJSON.module = `dist/${projectName}.esm.js`
```

Comment:
This should match what TSDX does, see here: https://github.com/formium/tsdx/blob/462af2d002987f985695b98400e0344b8f2754b7/src/utils.ts#L18-L21. It should also be used in the plugin "url" of any config file.

Reply:
Fixed